schemas/iso19139/src/main/plugin/iso19139/index-fields/index.xsl

```diff
@@ -344,6 +344,12 @@
           <xsl:for-each select="gmd:edition/*">
             <xsl:copy-of select="gn-fn-index:add-field('resourceEdition', .)"/>
           </xsl:for-each>
+
+          <xsl:for-each select="gmd:editionDate/*">
```

Comment:
Can you add a similar change in ISO19115-3.2008?

You can the following snippet:

```
<xsl:for-each select="cit:editionDate/*"> 
  <xsl:if  test="current() castable as xs:date or current() castable as xs:dateTime">
    <xsl:copy-of select="gn-fn-index:add-field('resourceEditionDate', .)"/>
  </xsl:if>
</xsl:for-each> 
 ```

below: https://github.com/geonetwork/core-geonetwork/blob/2249440b0582d09d661e8a93454872e880b4b82f/schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/index-fields/index.xsl#L393-L395